lib/rainbow.rb

```diff
@@ -11,11 +11,12 @@ def self.new
   self.enabled = false if ENV['TERM'] == 'dumb'
   self.enabled = true if ENV['CLICOLOR_FORCE'] == '1'
 
-  # On Windows systems, try to load the local ANSI support library
+  # On Windows systems, try to load the local ANSI support library if Ruby version < 2
+  # Ruby 2.x on Windows includes ANSI support. 
   require 'rbconfig'
   if RbConfig::CONFIG['host_os'] =~ /mswin|mingw/
     begin
-      require 'Win32/Console/ANSI'
+        require 'Win32/Console/ANSI' if RUBY_VERSION.to_i < 2
```

Comment:
Could we move this "if" 2 lines up, to make it `if RbConfig::CONFIG['host_os'] =~ /mswin|mingw/ && RUBY_VERSION.to_i < 2` ?